modules/nf-core/purecn/intervalfile/main.nf

```diff
@@ -0,0 +1,43 @@
+def VERSION = '2.4.0' // PureCN outputs to stderr instead of stdout, and exits with 1 with --version
+
+process PURECN_INTERVALFILE {
+    tag "${meta.id}"
+    label 'process_low'
+
+
+    conda "bioconda::bioconductor-purecn=2.4.0 bioconda::bioconductor-txdb.hsapiens.ucsc.hg38.knowngene=3.16.0 bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2 bioconda::bioconductor-org.hs.eg.db=3.16.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-582ac26068889091d5e798347c637f8208d77a71:a29c64a63498b1ee8b192521fdf6ed3c65506994-0':
+        'quay.io/biocontainers/mulled-v2-582ac26068889091d5e798347c637f8208d77a71:a29c64a63498b1ee8b192521fdf6ed3c65506994-0' }"
+
+    input:
+    tuple val(meta), path(target_bed)
+    path  fasta
+    val   genome
+
+    output:
+    tuple val(meta), path("*.txt"), emit: txt
+    // Only produced if --export is used
+    tuple val(meta), path("*.bed"), emit: bed, optional: true
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    """
+    library_path=\$(Rscript -e 'cat(.libPaths(), sep = "\\n")')
+    Rscript "\$library_path"/PureCN/extdata/IntervalFile.R --in-file ${target_bed} \\
+        --fasta ${fasta} \\
+        --out-file baits_intervals.txt \\
+        --genome ${genome} \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        purecn: ${VERSION}
+    END_VERSIONS
+    """
+}
```

Comment:
Can you add a stub section? [docs](https://www.nextflow.io/docs/latest/process.html#stub)